openmdao/docs/features/recording/case_reader.rst

```diff
@@ -0,0 +1,161 @@
+.. _case_reader:
+
+******************
+Case Reader Object
+******************
+
+The :class:`CaseRecorder<openmdao.recorders.case_reader.CaseReader>` object is provided to read case recordings no
+matter what case recorder was used.
+Currently, OpenMDAO only supports the :class:`SqliteCaseReader<openmdao.recorders.sqlite_reader.SqliteCaseReader>` case
+recorder. Therefore, all the examples will
+make use of this recorder. OpenMDAO will support other case recorders in the future.
+
+.. automethod:: openmdao.recorders.sqlite_reader.SqliteCaseReader.__init__
+    :noindex:
+
+Determining What Sources and Variables Were Recorded
+----------------------------------------------------
+
+The :code:`CaseReader` object provides methods to determine which objects in the original problem were responsible
+for recording cases and what variables they recorded.
+
+The :code:`list_sources` method provides a
+list of the names of objects that are the recording sources of the recorded data in the file.
+
+.. automethod:: openmdao.recorders.base_case_reader.BaseCaseReader.list_sources
+    :noindex:
+
+The complementary :code:`list_source_vars` method will provide a dictionary of the input and output variables recorded
+by each of those sources.
+
+.. automethod:: openmdao.recorders.base_case_reader.BaseCaseReader.list_source_vars
+    :noindex:
+
+Here is an example of their usage.
+
+.. embed-code::
+    openmdao.recorders.tests.test_sqlite_reader.TestFeatureSqliteReader.test_feature_list_sources
+    :layout: interleave
+
+
+Case Names
+----------
+
+A :code:`CaseReader` object contains a collection of :class:`Case<openmdao.recorders.case.Case>`
+objects, one for each individual recording of an object
+which can be anyone of these objects: driver, problem, component, or solver.
```

Comment:
This is not exactly true..  Case objects are actually created on the fly from the recorded data when a particular case is requested..  

Suggest something like "The CaseReader _provides access to_ Case objects, each of which encapsulates a data point recorded by one of the sources.  

Suggest moving the list of possible source types to the previous section relating to `list_sources`